composer.json

```diff
@@ -5,13 +5,12 @@
     "description": "Product bundle for Sylius.",
     "license": "MIT",
     "require": {
-        "php": "^7.2",
-
-        "sylius/sylius": "^1.4",
+        "php": "^7.3",
+        "sylius/sylius": "^1.7",
```

Comment:
For real not working on versions lower than 1.7?

Reply:
I was asked to update the plugin to the 1.7 version. I can make tests for older releases.

Reply:
Let's leave it as it is, but for future in case like that we should check on 1.4 if all works correctly we shouldn't exclude previous versions. Because updating to 1.7 not means from 1.7, but more means works also on 1.7.